libs/widget-lib/src/cowSwapWidget.ts

```diff
@@ -69,11 +70,14 @@ export function createCowSwapWidget(container: HTMLElement, props: CowSwapWidget
   let iframeRpcProviderBridge = updateProvider(iframeWindow, null, provider)
 
   // 7. Schedule the uploading of the params, once the iframe is loaded
-  iframe.addEventListener('load', () => updateParams(iframeWindow, params, provider))
+  iframe.addEventListener('load', () => updateParams(iframeWindow, currentParams, provider))
```

Comment:
Side fix.
Just noticed that parameter are being rewritten after some time.
We should use relevant params here, since they can be overridden in `updateParams`